### PR TITLE
1220 add stationmassa to windows hosts file

### PR DESCRIPTION
**Pull request by thomas-senechal**
## Delivery Quality Checklist

- [ ] **Breaking Changes in API:**
  Does this PR introduce breaking changes in the API?
    - If yes, have you considered making it backward compatible?
    - If backward compatibility is not considered, set the "breaking-change" label.

- [ ] **Changelog:**
  - [ ] For bugfix PR, set the "bugfix" label
  - [ ] If this change should not appear in changelog, use "ignore-for-changelog" label

- [ ] **Version Update Handling:**
  Have you ensured that the version update by user is handled correctly?

- [ ] **PR Dependency:**
  Does this PR depend on another PR?
    - If yes, is it necessary for the dependency to be released prior to merging this one?

- [ ] **Documentation:**
  - [ ] Are any necessary changes made to user-facing documentation?
  - [ ] Confirm that API documentation is updated with any relevant changes.
  - [ ] Check that README and other documentation files are accurate and current.

### CODE REVIEW

File: installer/windows/scripts/add_station_to_hosts.bat [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ECHO Executing add_station_to_hosts.bat
+
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+@REM Adds station.massa redirection to Windows hosts file if not already present
+
+SET windows_hosts_file=%windir%\System32\drivers\etc\hosts
+
+@REM If station.massa is already in Windows hosts file, we can exit
+FINDSTR /c:"127.0.0.1 station.massa" "%windows_hosts_file%" >nul 2>&1
```
Comment (gregLibert) [inline installer/windows/scripts/add_station_to_hosts.bat:12]: Can you only check for "station.massa" to handle more complex nework scenarios ?

File: installer/windows/scripts/add_station_to_hosts.bat [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+if %errorlevel%==0 (
+    ECHO TLD already in the file
```
Comment (gregLibert) [inline installer/windows/scripts/add_station_to_hosts.bat:14]:
```suggestion
    ECHO station.massa host already configured
```

File: installer/windows/scripts/add_station_to_hosts.bat [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@ECHO OFF
+
+ECHO Executing add_station_to_hosts.bat
+
+SETLOCAL ENABLEDELAYEDEXPANSION
+
+@REM Adds station.massa redirection to Windows hosts file if not already present
```
Comment (gregLibert) [inline installer/windows/scripts/add_station_to_hosts.bat:7]:
```suggestion
@REM Adds station.massa host to Windows hosts file if not already present
```

**Review by gregLibert — CHANGES_REQUESTED** [review]
The host configuration is not removed during uninstall

**Review by gregLibert — CHANGES_REQUESTED** [review]
The host configuration is not removed during uninstall

File: installer/windows/scripts/add_station_to_hosts.bat (hunk 1 above)
Comment (gregLibert) [reply installer/windows/scripts/add_station_to_hosts.bat:12]: Not fixed:

image

**Review by gregLibert — APPROVED** [review]

**Review by peterjah — APPROVED** [review]

**Pull request merged**